shippable.yml

```diff
@@ -38,6 +38,7 @@ matrix:
     - env: T=linux/centos8/1
     - env: T=linux/fedora30/1
     - env: T=linux/fedora31/1
+    - env: T=linux/fedora32/1
```

Comment:
Go ahead and remove F30 from CI now since it goes EOL soon. We've only been keeping the 2 latest Fedora releases in the CI matrix.